presto-druid/pom.xml

```diff
@@ -13,6 +13,7 @@
 
     <properties>
         <air.main.basedir>${project.parent.basedir}</air.main.basedir>
+        <dep.log4j.version>2.16.0</dep.log4j.version>
```

Comment:
Quick question: Any specific reasons that you want to explicitly add log4j to the presto-druid connector? Is the druid connector using log4j?
Cc @zhenxiao and @beinan as it may have an impact on the Twitter's presto use case.

Reply:
druid packages that the connector imports depend directly on log4j so log4j will be brought in, alternatively we can forego this and upgrade druid, but requires more conflict resolution with other libraries. 
By adding a dependency to the good log4j will force maven to get the latest, overriding the one imported by this version of drui..

Same thing for presto-elasticsearch. 

Open to alternative suggestions and there's room for a larger discussion how we deal with these dependencies -- this approach is not the best at presto scale.

for now I did what appeared the minimal change so we can release 0.267 without publishing a vulnerable maven package.

Reply:
LGTM. Thanks for your clarification!